src/components/ReportActionItem/RenameAction.js

```diff
@@ -5,16 +5,22 @@ import Text from '../Text';
 import styles from '../../styles/styles';
 import reportActionPropTypes from '../../pages/home/report/reportActionPropTypes';
 import withLocalize, {withLocalizePropTypes} from '../withLocalize';
+import compose from '../../libs/compose';
+import withCurrentUserPersonalDetails, {withCurrentUserPersonalDetailsPropTypes} from '../withCurrentUserPersonalDetails';
 
 const propTypes = {
     /** All the data of the action */
     action: PropTypes.shape(reportActionPropTypes).isRequired,
 
     ...withLocalizePropTypes,
+    ...withCurrentUserPersonalDetailsPropTypes,
 };
 
 const RenameAction = (props) => {
-    const displayName = lodashGet(props.action, ['message', 0, 'text']);
+    const currentUserLogin = props.currentUserPersonalDetails.login;
+    const userDisplayName = lodashGet(props.action, ['person', 0, 'text']);
+    const actorEmail = lodashGet(props.action, 'actorEmail', '');
+    const displayName = actorEmail === currentUserLogin ? `${props.translate('common.you')}` : `${userDisplayName}`;
```

Comment:
This check is breaking when I change my contact method to secondary email or phone number.

<img width="1084" alt="Screenshot 2023-06-10 at 8 28 31 PM" src="https://github.com/Expensify/App/assets/46707890/717955f8-01fa-470b-ab1c-8f0caf2081fb">
<img width="1303" alt="Screenshot 2023-06-10 at 8 29 31 PM" src="https://github.com/Expensify/App/assets/46707890/1e58f4a1-eecc-4f03-8745-f5623ecb4c78">

We have to check with account if the current accountID is same so that of the action, this way even after contact methods changes it will be shown correctly. If accountID information is not available in this case, we can park this change for now since its a extra change for this PR and we are having an initiative to start using accountIDs instead of login checks.

Reply:
I don't think that will work as well. Since the last actor email is the email with which the user took the action. If the user changed the profile email to secondary email it won't match is my assumption. Correct me if wrong.

Reply:
@Ollyws do check and let us know.